java/images/jboss/s2i/assemble

```diff
@@ -160,14 +189,59 @@ function build_maven() {
   cd ${old_dir}
 }
 
+function build_gradle() {
+  # Where artifacts are created during build
+  local build_dir=$1
+
+  # Where to put the artifacts
+  local app_dir=$2
+
+  # Default args
+  local gradle_args=${GRADLE_ARGS:-build -x test}
+
+  # If there is no user provided GRADLE_OPTS, use options from run-java.sh
+  if [ -f ${RUN_JAVA_DIR}/run-java.sh -a -z "${GRADLE_OPTS}" ]; then
```

Comment:
using `[ .... ] && [ .... ]` proved to be more robust to me. `-a` fails in certain edge case (which I dont remember right now ;-)